code/go/0chain.net/smartcontract/minersc/handler.go

```diff
@@ -58,9 +58,105 @@ func GetEndpoints(rh rest.RestHandlerI) []rest.Endpoint {
 		rest.MakeEndpoint(miner+"/configs", common.UserRateLimit(mrh.getConfigs)),
 		rest.MakeEndpoint(miner+"/get_miner_geolocations", common.UserRateLimit(mrh.getMinerGeolocations)),
 		rest.MakeEndpoint(miner+"/get_sharder_geolocations", common.UserRateLimit(mrh.getSharderGeolocations)),
+		rest.MakeEndpoint(miner+"/provider-rewards", common.UserRateLimit(mrh.getProviderRewards)),
+		rest.MakeEndpoint(miner+"/delegate-rewards", common.UserRateLimit(mrh.getDelegateRewards)),
 	}
 }
 
+// swagger:route GET /v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d7/delegate-rewards delegate-rewards
+// Gets list of delegate rewards satisfying filter
```

Comment:
please use the minerSC REST endpoint URL
`v1/screst/6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712d9/delegate-rewards`

Reply:
I don't understand, you are saying change so that the endpoints prefix is the storagesc address rather than the minersc address. In which case we should move the endopints to storagesc not just change the endpoint address. 

This endpoint covers data from all providers. At some point, we should have a new address to cover handlers that service all providers. But that is beyond the scope of this PR.